docs/CONNECTOR_PROTOCOL.md

```diff
@@ -276,7 +290,10 @@ In addition to the connector index `.elastic-connectors`, we have an additional
     "version" : 1
   },
   "properties" : {
+    "cancelation_requested_at" : { "type" : "date" },
```

Comment:
I wonder why you need this property. Could you please share more information about that?

Reply:
This is explained above:
>cancelation_requested_at: date; -> The date/time when the cancelation of the job is requested
  canceled_at: date; -> The date/time when the job is canceled

It's the date/time when an operator clicks `Cancel` in Kibana. `canceled_at` is the date/time the job is actually canceled by the connector.

Free feel to propose a new name.

Reply:
Thanks. My questions was about where are you going to use that property? Is it gonna be used in the code? Or maybe in Kibana UI?

Reply:
It's not going to be used in the connector code. I think just for audit purposes, that we know when it's requested. And I think this will also be used in Kibana to display events, until we have events generator, similar to Crawler.

Reply:
I would not add it if it's not required. This really sounds like a good use case for logging. 

I wonder if we can retrieve this information from Kibana server log for example.

Reply:
>I would not add it if it's not required.

It's required, see [here](https://docs.google.com/document/d/1mqPTc3Agg7FaUPVZ0YR4_R3M552HjqFwAL1UJvi57Ek/edit#bookmark=id.gd6hz8otbhqh). I meant it's not used by backend code (It's not set by BE, it's not used by BE either).